### DISCUSSION TITLE
Getting exception in Autogen.Net

### DISCUSSION THREAD

**Discussion by teeshajain17** [microsoft/autogen#2972] [Q&A]
I was running this example in my system and it is throwing error.
https://github.com/microsoft/autogen/blob/ca4f7176169d5f40b5e7570aab7dce4a37bd088f/dotnet/sample/AutoGen.BasicSamples/Example04_Dynamic_GroupChat_Coding_Task.cs
![image](https://github.com/microsoft/autogen/assets/16876986/7d6b282a-f41e-4c61-b124-d2d13fc155ec)

**Comment by LittleLittleCloud** [comment]
#2970
